python/setup.py

```diff
@@ -4,17 +4,13 @@
 
 import subprocess
 
-from setuptools import setup, find_packages
-import setuptools.command.install as _install
+from setuptools import setup, find_packages, Extension, Distribution
+import setuptools.command.build_ext as _build_ext
 
 
-class install(_install.install):
+class build_ext(_build_ext.build_ext):
   def run(self):
     subprocess.check_call(["../build.sh"])
```

Comment:
Looks like we also need to manually copy all of the files over somewhere. e.g., https://github.com/apache/arrow/blob/master/python/setup.py#L218